### PR TITLE
Version compatible to stdlib 1.7 and agda 2.6.2

### PR DESCRIPTION
**Pull request by Akshobhya1234**
**PR changing the CI variable to use 2.6.2 and 1.7**
Issue #287

### CODE REVIEW

File: .github/workflows/ci-ubuntu.yml
Comment (gallais) [inline .github/workflows/ci-ubuntu.yml:48]:
```suggestion
  AGDA_COMMIT: tags/v2.6.2
```

**Comment by ice1000** [comment]
image

GitHub is not showing you as the author of these commits. Maybe you should reconfigure your email settings.

**Comment by ice1000** [comment]
Let's run the GitHub workflow

**Comment by Akshobhya1234** [comment]
> image
> 
> GitHub is not showing you as the author of these commits. Maybe you should reconfigure your email settings.

"akshobhya.k.m@sap.com" is my enterprise account. "Akshobhya1234" is my personal account. My local is setup for enterprise account and I have forked the repo to my personal account.

**Comment by ice1000** [comment]
> > GitHub is not showing you as the author of these commits. Maybe you should reconfigure your email settings.
> 
> "[akshobhya.k.m@sap.com](mailto:akshobhya.k.m@sap.com)" is my enterprise account. "Akshobhya1234" is my personal account. My local is setup for enterprise account and I have forked the repo to my personal account.

I totally understand the reason why this happens. I'm just saying that by doing this, GitHub will not count you, the user @Akshobhya1234, as a contributor to this project.

**Comment by Akshobhya1234** [comment]
> > > GitHub is not showing you as the author of these commits. Maybe you should reconfigure your email settings.
> > 
> > 
> > "[akshobhya.k.m@sap.com](mailto:akshobhya.k.m@sap.com)" is my enterprise account. "Akshobhya1234" is my personal account. My local is setup for enterprise account and I have forked the repo to my personal account.
> 
> I totally understand the reason why this happens. I'm just saying that by doing this, GitHub will not count you, the user @Akshobhya1234, as a contributor to this project.

Yes. But once this PR is merged I think it will consider with @Akshobhya1234  as contributor ( based on merge commit). It is fine for me if GitHub doesn't count me as contributor as long as it is fine with the team.

**Review by gallais — APPROVED** [review]

**Comment by ice1000** [comment]
@gallais merge?

**Review by JacquesCarette — APPROVED** [review]

**Comment by JacquesCarette** [comment]
Thanks, merged. I'm going to wait for the build to finish to make a release.

**Pull request merged**
